crates/astria-conductor/src/executor.rs

```diff
@@ -149,16 +149,19 @@ impl<C: ExecutionClient> Executor<C> {
                 ExecutorCommand::BlockReceivedFromGossipNetwork {
                     block,
                 } => {
+                    let height = block.header().height.value();
                     self.alert_tx.send(Alert::BlockReceivedFromGossipNetwork {
-                        block_height: block.header().height.value(),
+                        block_height: height,
                     })?;
-                    if let Err(e) = self
-                        .execute_block(SequencerBlockSubset::from_sequencer_block_data(
-                            *block,
-                            self.namespace,
-                        ))
-                        .await
-                    {
+
+                    let Some(subset) =
+                        SequencerBlockSubset::from_sequencer_block_data(*block, self.namespace)
+                    else {
```

Comment:
What is this else on?

Reply:
it's `let Some(..) else` syntax: https://doc.rust-lang.org/rust-by-example/flow_control/let_else.html